test/magnetic-test.js

```diff
@@ -0,0 +1,270 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('./util/assert');
+const common = require('../lib/script/common');
+const Script = require('../lib/script/script');
+const Stack = require('../lib/script/stack');
+const Keyring = require('../lib/primitives/keyring');
+const secp256k1 = require('bcrypto/lib/secp256k1');
+const sha256 = require('bcrypto/lib/sha256');
+
+const EMPTY = Buffer.alloc(0);
+
+let flags = [
+  common.flags.VERIFY_NONE,
+  common.flags.STANDARD_VERIFY_FLAGS,
+  common.flags.MANDATORY_VERIFY_FLAGS
+];
+
+// TODO: update standard flags
+flags = flags.map(flag => flag & ~common.flags.VERIFY_COMPRESSED_PUBKEYTYPE);
+
+const priv = '0000000000000000000000000000000000000000000000000000000000000001';
+const key = Keyring.fromPrivate(Buffer.from(priv, 'hex'));
+
+function isSuccess(stack, script, expected) {
```

Comment:
Add `flags` as param instead of global var.

Reply:
Let's do this after release :+1: